quadratic_equation.h

```diff
@@ -3,57 +3,5 @@
 #include "assert.h"

```

Comment:
This include is used only in quadratic_equation.c, which is why it's better to write it there.